test/functional/cmdLineTests/gcRegressionTests/playlist.xml

```diff
@@ -44,6 +44,36 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_GCRotatingVerboseLogTests</testCaseName>
+		<variations>
```

Comment:
Please add NoOptions to cover `cmdLineTester_GCRegressionTests_RISCV` case.

Reply:
NoOptions has been added too.